cwl/fingerprint.cwl

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: Workflow
+doc: "
+Workflow to run generate fingerprint search on entire dmp normals and optionally on project normals
+"
+
+requirements:
+  MultipleInputFeatureRequirement: {}
+  ScatterFeatureRequirement: {}
+  StepInputExpressionRequirement: {}
+  InlineJavascriptRequirement: {}
+  SubworkflowFeatureRequirement: {}
+
+hints:
+  cwltool:LoadListingRequirement:
+    loadListing: no_listing
+
+inputs:
+  dmp_dir:
+    type: Directory
+    default:
+      class: Directory
+      path: /work/ci/dmp_finderprint_matching/dummy_pileup
+  conpair_markers_bed:  # conpair_markers_txt
+    type: File
+  conpair_markers_txt:  # conpair_markers_txt
+    type: File
+  tumor_bam: # GATK .pileup or likelihoods .pickle
+    type: File
+    secondaryFiles: ["^.bai"]
+  additional_normal_bams:
+    type: File[]
+    secondaryFiles: ["^.bai"]
+    default: [{class: File, path: /work/ci/dmp_finderprint_matching/dummy_bam/dummy.bam}]
+  ref_fasta:
+    type: File
+    secondaryFiles:
+      - .amb
+      - .ann
+      - .bwt
+      - .pac
+      - .sa
+      - .fai
+      - ^.dict
+
+
+outputs:
+  output_file:
+    type: File
+    outputSource: run_conpair_concordance/output_file
+
+steps:
+  run-pileup-tumor:
+    run: conpair-pileup.cwl
+    in:
+      bam: tumor_bam
+      ref: ref_fasta
+      gatk:
+        valueFrom: ${ return '/usr/bin/gatk.jar'; }
+      markers_bed: conpair_markers_bed
+      java_xmx:
+        valueFrom: ${ return ["24g"]; }
```

Comment:
this is not a default setting?
this is a memory requirement so the CWL will also need a memory requirement